src/stream.c

```diff
@@ -65,6 +65,7 @@ struct stream {
 	struct menc_media *mes;  /**< Media Encryption media state          */
 	enum media_type type;    /**< Media type, e.g. audio/video          */
 	char *cname;             /**< RTCP Canonical end-point identifier   */
+	char *peer;              /**< RTCP Canonical end-point identifier   */
```

Comment:
pls update Doxygen!

Reply:
@cspiel1 , I'm not entirely clear about the request - should I change the contents of the comments to reflect something specific or run a doxygen command to generate update documentation?